dbms/src/Storages/DeltaMerge/DMSegmentThreadInputStream.h

```diff
@@ -16,6 +16,7 @@
 
 #include <Common/FailPoint.h>
 #include <DataStreams/IProfilingBlockInputStream.h>
+#include <Interpreters/Context.h>
 #include <Interpreters/Context_fwd.h>
```

Comment:
```suggestion
#include <Interpreters/Context_fwd.h>
```
The `Context.h` could be removed because we have already include `Context_fwd.h`